CMakeLists.txt

```diff
@@ -98,6 +99,14 @@ macro(find_package_write_status pkg)
   endif()
 endmacro()
 
+#===============================================================================
+# NCrystal Scattering Support
+#===============================================================================
+
+if(OPENMC_USE_NCRYSTAL)
+  find_package(NCrystal REQUIRED PATH_SUFFIXES ./)#fixme
```

Comment:
Concerning this fixme: We were not sure about the `PATH_SUFFIXES ./` part. We saw `PATH_SUFFIXES` being used elsewhere in the file, but normally we would expect a `find_package(NCrystal)` to be sufficient.